### ISSUE TITLE
adding an HTTP link to a step

### ISSUE THREAD

**Issue by adrian-herscu** [TNG/JGiven#1356]
Besides attachments, is it possible to add some arbitrary HTML to a step?
For example an HTTP link to a step?

I would like for certain steps, namely Given step that establishes a Selenium session, to add a link for a video recording of that session, for example on SauceLabs.

**Comment by l-1squared** [comment]
Hi @adrian-herscu,
if I recall correctly, we handle attachments more or less as arbitrary bytes, although we might make some assumptions about the possible MIME types. I'd need to look into the code exactly.

Please keep in mind that it is holiday season, so response might be a bit slow at the moment.

**Comment by l-1squared** [comment]
short answer ( I haven't reviewed your request fully); but you should at least be able to attach videos. there is a corresponding media type  in the `MediaType.java`

   /**
     * Creates a binary video media type with the given subtype.
     */
    public static MediaType video( String subType ) {
        return binary( VIDEO, subType );
    }

**Comment by adrian-herscu** [comment]
Yes, saw that... As far as I understand it means that the video will be
embedded. That's not good. The report file will be very big and there will
be no controls. I just want a link to the video as recorded by an external
Selenium provider (e.g. SauceLabs).
